doc/rst/source/grdmix.rst

```diff
@@ -0,0 +1,181 @@
+    The final image will have a transparency layer determined by these values.
+
+.. _-C:
+
+**-C**
+    **C**\ onstruct an output image from one or three normalized input grids;
+    these grids must all have values in the 0-1 range only (see **-N** if they don't).
+    Optionally, use **-A** to add transparency and **-I** to add intensity
+    to the colors before writing the image. For three layers the input order must
+    be red grid first, then the green grid, and finally the blue grid.
+
+.. _-D:
+
+**-D**
+    **D**\ econstruct a single image into one or three normalized output grids.
+    An extra grid will be written if the image contains an alpha (transparency layer).
+    All grids written will have values in the 0-1 range exclusively; however, you
+    can use **-N** to undo this implicit normalization.
+    The output names uses the name template given by **-G** which must contain the
+    C-format string "%c".  This code is replaced by the codes R, G, B and A for color
+    images and g, A for gray-scale images.
+
+.. _-I:
+
+**-I**\ *intensity*
+    A constant intensity or grid (-1/+1 range) to modify final output image colors.
+
+.. _-M:
+
+**-M**
+    Force conversion to monochrome image using the (television) YIQ
+    transformation.
+
+.. _-N:
+
+**-N**\ [*factor*] 
+    Normalize all input grids from 0-255 to 0-1 [All input grids are assumed to be in 0-1 range].
+    To normalize by another factor than 255, append an optional *factor* value.
+
+.. _-Q:
+
+**-Q**
+    Make the final image opaque by removing the alpha layer (if present).
+
+.. _-R:
+
+.. |Add_-R| unicode:: 0x20 .. just an invisible code
+.. include:: explain_-R.rst_
+
+.. _-V:
+
+.. |Add_-V| unicode:: 0x20 .. just an invisible code
+.. include:: explain_-V.rst_
+
+.. _-W:
+
+**-W**\ *weights*
+    A constant weight (0-1), or a grid (0-1) or image (0-255) with weights.
+    When two input rasters are given, the weights are applied to *raster1* and
+    (*1-weights*) are applied to *raster2*, then the products are summed.
+
+.. |Add_-f| unicode:: 0x20 .. just an invisible code
+.. include:: explain_-f.rst_
+
+.. include:: explain_help.rst_
+
+.. include:: explain_grd_inout_short.rst_
+
+Examples
+--------
+
+.. include:: explain_example.rst_
+
+To blend the night and day views of the Earth using a weight image computed for
+a particular day/night terminus, try::
+
+    gmt grdmix @BlueMarble_06m.tif @BlackMarble_06m.tif -W@weight.png -Gnewmap.png
+
+Suppose map1.png and map2.png are overlapping maps of different quantities, but we wish
+to use the image visible.png to blend them into a single image: Where visible.png has
+values close to 255 we will see predominantly the map1.png contents while for values
+closer to zero we will mostly see map2.png - values in between these extremes will
+lead to a weighted average.  We try::
+
+    gmt grdmix map1.png map2.png -Avisible.png -Gnewmap.png -V
```

Comment:
Is this example still valid? Are you talking about **-W** or **-A**?

Reply:
Sorry, you are right, this is now -Q.  Fixed, plus a typo further down,